tests/test_bs_helper.py

```diff
@@ -43,19 +42,19 @@ def setUp(self, mocked_soup):
         self.mocked_soup = mocked_soup
         self.mocked_soup.select_one.return_value = 'some text'
         self.selector = 'some_tag'
+        self.expected_text = 'some_tag1', 'some_tag2', 'some_tag3'
 
     def test_bs_safe_select_return_expected_text_with_single_selector(self):
         actual = self.source_page.bs_safe_select(self.mocked_soup,
                                                  self.selector)
         self.assertEqual(actual, 'some text')
 
     def test_bs_safe_select_return_expected_text_with_many_selectors(self):
-        selector1, selector2, selector3 = 'some_tag1', 'some_tag2', 'some_tag3'
-        actual = self.source_page.bs_safe_select(self.mocked_soup, selector1,
-                                                 selector2, selector3)
+        actual = self.source_page.bs_safe_select(self.mocked_soup,
+                                                 self.expected_text)
```

Comment:
previously you were passing selectors, now expected text. I see that it is the same, then it might be wrong naming